ops/storage-ddls/cassandra-setup.cql

```diff
@@ -165,9 +165,8 @@ CREATE TABLE computedtiles (
     tiley int,
     externalsourceid text,
     mentioncount bigint,
-    avgsentiment float,
+    avgsentimentnumerator bigint,
```

Comment:
There's a related change necessary in the materialized view that depends on this table. Would just remove these changes from the PR, keep it focused on the storage refactor and instead merge https://github.com/CatalystCode/project-fortis-pipeline/pull/118 for the CQL change.